swift/apple/FirezoneKit/Sources/FirezoneKit/Helpers/IPCClient.swift

```diff
@@ -260,4 +295,17 @@ class IPCClient {
 
     throw Error.invalidStatus(session.status)
   }
+
+  private func sendMessageWithoutResponse(_ message: ProviderMessage) async throws {
+    try await withCheckedThrowingContinuation { continuation in
+      do {
+        try session().sendProviderMessage(encoder.encode(message)) { _ in
+          continuation.resume()
+        }
+      } catch {
+        Log.error(error)
```

Comment:
Are we sure this isn't double logged?

Reply:
Catch only happens once

Reply:
You are essentially rethrowing the error, that is what I mean :)

Isn't the caller likely going to catch the exception and throw again as well?

Reply:
Ah yes, good catch. I'm not sure if/where we log this, but yes we should bubble errors up as far as possible since we may need to show the user them. I'll open a fix.